hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -148,6 +150,11 @@ def _default_employee(self):
         compute='_compute_total_time',
         store=True,
     )
+    _dummy_field = fields.Boolean(
+        compute='_compute_dummy_field',
+        store=False,
+        # hack: needed to force recompute "stored" lines
```

Comment:
But if the model is not transient, it will occupy a lot of unnecessary data in the database.

Reply:
What about `unlink()` ?

Reply:
Anyway when I was working on this module I noticed that there were never leftover records for the lines, because the system automatically removed the orphan lines when recomputing. Not sure that now the behavior is different because of the `stored=True`. Would you give it a try? Maybe the `unlink()` is not even necessary.

Reply:
You will only unlink the deprecated lines, but you will not unlink the lines used for all existing sheets. And in a year of sheets for just an employee, already it's a lot of lines. I don't want to imagine how many lines this suppose for several employees :S

Reply:
@mreficent sorry I don't understand your point. At the moment you create new lines you can also delete old obsolete lines. And this is not even necessary in case the system does it automatically for you.

Reply:
What do you mean? I have a sheet for this week with 4 lines. Yeah, I can change this 4 rows for other different 4 rows. Unlink ones and create others. But, this sheet will always have 4 rows (and thus 4x7 lines occupying space in database).

Reply:
Yes, and what's the problem then? You made them stored, this is what are you proposing. Next time you open the timesheet sheet you will see the same lines again without need of recomputing.

Reply:
I prefer not storing the lines and recomputing them when opening a sheet form. I put the "store=True" knowing they are transient. The need of the "store=True" was to fix some bad behavior when you add rows in the summary using the add project button.